db/schema.rb

```diff
@@ -54,6 +54,17 @@
     t.index ["profile_id"], name: "index_documents_on_profile_id"
   end
 
+  create_table "labels", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+    t.string "key", null: false
+    t.string "value", null: false
+    t.integer "scope", default: 0, null: false
+    t.bigint "account_id"
```

Comment:
put account_id at the beginning of the table.

Reply:
@mod sure

Reply:
done